starkit/gridkit/base.py

```diff
@@ -309,6 +311,15 @@ def load_grid(hdf_fname, wavelength_type=None, base_class=BaseSpectralGrid):
     spec_grid = SpectralGrid(wavelength, index[meta['parameters']].values, fluxes, meta,
                              wavelength_type=wavelength_type,  **initial_parameters)
 
+    if 'format_version' not in spec_grid.meta_grid.keys():
+        logger.warn('No format_version in meta data for this grid. Please get an updated grid. This will fail in the future.')
+        warnings.warn('No format_version in meta data for this grid. Please get an updated grid. This will fail in the future.', DeprecationWarning)
+    else:
+        current_version = starkit.gridkit.FORMAT_VERSION
+        grid_version = spec_grid.meta_grid['format_version']
+        if current_version[1] != grid_version[1]:
+            raise ValueError('Grid major versions do not match! Curent code format version: '+current_version+' grid format version: '+grid_version)
```

Comment:
please use `.format()`. 
```suggestion
            raise ValueError('Grid major versions do not match! Curent code format version: {0} grid format version: {1} '.format(current_version, grid_version))
```

Reply:
and line too long